x-pack/plugins/transform/public/app/common/validators/frequency_validator.test.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { frequencyValidator } from './frequency_validator';
+
+describe('Transform: frequencyValidator()', () => {
+  const transformFrequencyValidator = (arg: string) => frequencyValidator(arg).length === 0;
```

Comment:
Not sure if we should define a function inside of a test suite and use it for assertion 🤔 
Usually, you test a function that is used in runtime.

Reply:
You're right, I removed it in here [0e8ab71](https://github.com/elastic/kibana/pull/173736/commits/0e8ab71711b11ae83d836066592f51e22bfd2216), it's now in line with the other test files.